cdap-docs/developers-manual/source/building-blocks/datasets/index.rst

```diff
@@ -119,8 +120,10 @@ the name of that column is a metadata property of each Dataset of this type.
 
 .. rubric:: Core Datasets
 
-**Tables** are the only core Datasets, and all other Datasets are built using one or more
-Tables. These Tables are similar to tables in a relational database with a few key differences:
+**Tables** and **FileSets** are the only core Datasets, and all other Datasets are built using
+one or more Tables or FileSets.
+
+While these Tables have rows and columns relational database tables, there are a few key differences:
```

Comment:
While these Tables have rows and columns similar to relational database tables, there are key differences: